scripts/geowaveTestDB.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+wget https://s3.amazonaws.com/geotrellis-test/geowave-minicluster/miniAccumuloCluster-assembly-0.1.0.jar
+
+java -jar miniAccumuloCluster-assembly-0.1.0.jar
```

Comment:
This testing solution is *far* from optimal, but it works and avoids the considerable headaches introduced by other workarounds for this integrationy unit test